src/cephadm/cephadm.py

```diff
@@ -4678,6 +4680,16 @@ def probe_hba(scan_path: str) -> None:
 
     return f'Ok. {len(all_scan_files)} adapters detected: {len(scan_files)} rescanned, {len(skipped)} skipped, {len(failures)} failed ({elapsed:.2f}s)'
 
+def command_list_images(ctx: CephadmContext) -> None:
+    """this function will list the default images used by different services"""
+    cp_obj = ConfigParser()
+    cp_obj['mgr'] = get_mgr_images()
+    # print images
+    for section in cp_obj.sections():
+        print(f"[{section}]")
+        for key, value in cp_obj.items(section):
+            print(f"{key} = {value}")
+        print()
```

Comment:
the ConfigParser object has a write method that basically does all of this for you:
```
>>> from configparser import ConfigParser
>>> import sys
>>> cp_obj = ConfigParser()
>>> cp_obj['mgr'] = {'foo': 'foo.org/mystuff/image:default', 'bar': 'bar.io/iron/steel:1.44'}
>>> cp_obj.write(sys.stdout)
[mgr]
foo = foo.org/mystuff/image:default
bar = bar.io/iron/steel:1.44
```

Reply:
Thanks for pointing this. I have updated the code.